src/components/screen/SignUp.tsx

```diff
@@ -0,0 +1,193 @@
+        <Wrapper>
+          <EditText
+            testID="input-email"
+            errorTestID="error-email"
+            textStyle={{
+              color: theme.fontColor,
+            }}
+            label={getString('EMAIL')}
+            placeholder="hello@example.com"
+            placeholderTextColor="#ADB5BD"
+            value={email}
+            onChangeText={(text: string): void => {
+              setEmail(text);
+              setErrorEmail('');
+            }}
+            style={{ marginTop: 20 }}
+            errorText={errorEmail}
+            onSubmitEditing={onSignUp}
+          />
+          <EditText
+            testID="input-password"
+            errorTestID="error-password"
+            textStyle={{
+              color: theme.fontColor,
+            }}
+            label={getString('PASSWORD')}
+            value={password}
+            onChangeText={(text: string): void => {
+              setPassword(text);
+              setErrorPassword('');
+            }}
+            style={{ marginTop: 20 }}
+            errorText={errorPassword}
+            onSubmitEditing={onSignUp}
+            secureTextEntry={true}
+          />
+          <EditText
+            testID="input-confirm-password"
+            errorTestID="error-confirm-password"
+            textStyle={{
+              color: theme.fontColor,
+            }}
+            label={getString('CONFIRM_PASSWORD')}
+            value={confirmPassword}
+            onChangeText={(text: string): void => {
+              setConfirmPassword(text);
+              setErrorConfirmPassword('');
+            }}
+            style={{ marginTop: 20 }}
+            errorText={errorConfirmPassword}
+            onSubmitEditing={onSignUp}
+            secureTextEntry={true}
+          />
+          <EditText
+            testID="input-name"
+            errorTestID="error-name"
+            textStyle={{
+              color: theme.fontColor,
+            }}
+            label={getString('NAME')}
+            placeholder="Write email address"
+            placeholderTextColor="#ADB5BD"
+            value={name}
+            onChangeText={(text: string): void => {
+              setName(text);
+              setErrorName('');
+            }}
+            style={{ marginTop: 20 }}
+            errorText={errorName}
+            onSubmitEditing={onSignUp}
+          />
+          <EditText
+            testID="input-status"
+            errorTestID="error-status"
+            textStyle={{
+              color: theme.fontColor,
+            }}
+            label={getString('STATUS')}
+            placeholder="Write status"
+            placeholderTextColor="#ADB5BD"
+            value={status}
+            onChangeText={(text: string): void => {
+              setStatus(text);
+            }}
+            style={{ marginTop: 20 }}
+            onSubmitEditing={onSignUp}
+          />
+          <ButtonWrapper>
+            <View style={{ flex: 1 }}/>
```

Comment:
Is this necessary??

Reply:
Please check [this](https://developer.mozilla.org/ko/docs/Web/CSS/flex-direction).